src/aria/widgets/container/Dialog.js

```diff
@@ -580,9 +580,45 @@ module.exports = Aria.classDefinition({
             var cfg = this._cfg;
             var getDomElementChild = ariaUtilsDom.getDomElementChild;
             this._domElt = this._popup.domElement;
-            this._titleBarDomElt = getDomElementChild(this._domElt, 0, true);
-            this._titleDomElt = getDomElementChild(this._titleBarDomElt, cfg.icon ? 1 : 0);
+            var titleBarDomElt = this._titleBarDomElt = getDomElementChild(this._domElt, 0, true);
+            var bodyElement = getDomElementChild(this._domElt, 1, true);
```

Comment:
I would do: `var bodyElement = getDomElementChild(this._domElt, 0);` because the body element is not always the element before the last element inside `this._domElt`, especially if the dialog is resizable (cf https://github.com/fbasso/ariatemplates/blob/dialog_maxwidth/src/aria/widgets/container/Dialog.js#L315). However, it should always the first element element in `this._domElt`.